src/h5web/visualizations/heatmap/hooks.ts

```diff
@@ -26,20 +27,14 @@ export interface TextureDataState {
 export function useTextureData(
   rows: number,
   cols: number,
-  values: number[]
+  values: number[],
+  domain: Domain | undefined
 ): TextureDataState {
-  const [dataDomain, customDomain, scaleType, colorMap] = useHeatmapConfig(
-    (state) => [
-      state.dataDomain,
-      state.customDomain,
-      state.scaleType,
-      state.colorMap,
-    ],
+  const [scaleType, colorMap] = useHeatmapConfig(
+    (state) => [state.scaleType, state.colorMap],
```

Comment:
Might as well pass everything through the props now, don't you think?

Reply:
I think therefore I will do so :slightly_smiling_face: